scripts/releases.py

```diff
@@ -73,14 +73,17 @@ def main():
     parser.add_argument(
         "--release-date", default=None, help="Date of the release")
     parser.add_argument(
-        "--db-size", default=None, help="Size of the database for the release in GB")
+        "--db-size", default=None,
+        help="Size of the database for the release in GB")
     parser.add_argument("--format", default="tsv", help=(
         "Output format, includes 'string', 'csv', 'tsv' (default), and "
         "'json'. "
-        "'tsv' can be appended to the IDR studies.csv file with no further "
-        "processing. "
         "All other formats include headers and totals. "
         "'string' is the most human-readable (fixed width columns)."
+        "If tsv is selected and a file called releases.tsv exists in the "
+        "same directory as the file specified by studies_file, the output "
+        "will be appended to this file."
+        "'string' is the most human-readable (fixed width columns)."
```

Comment:
```suggestion
```

This seems to be a repetition of https://github.com/IDR/idr-utils/pull/19/commits/eebda2c24183e573668da9468ab391a0e90f0418#diff-a20aed2e870f7cfe8de9cd8f4c2fa0c6R82 above, so suggesting a removal.